index.md

```diff
@@ -57,36 +57,40 @@ The Sparkbox Apprenticeship is loosely separated into three phases. Completing a
 * [Game of Life Kata - Ruby][]
 * [FizzBuzz Kata - Ruby][]
 * [Bowling Kata - Ruby][]
-* Tic-tac-toe Kata - Ruby
+* [Tic-tac-toe Kata - Ruby](https://gist.github.com/dlresende/dc57f506e1ab1de7e7df)
 
 **Training:**
 
-* Introduction to frontend project
-* [CodeSchool: Front-end Foundations][]: Level 2, Level 3, Level 5
+* Frontend Project: Introduction
```

Comment:
Let's split out the changes to frontend from the rest. I want to talk with @zastrow about how these should be aligned with the milestones, if at all.